backend/app/serializers.py

```diff
@@ -62,3 +63,15 @@ class CorpusSerializer(serializers.ModelSerializer):
     class Meta:
         model = Corpus
         fields = ['id', 'title', 'description', 'documents']
+
+
+class ProximitySerializer(serializers.ModelSerializer):
```

Comment:
Should this be `ProximityAnalysisSerializer`?

Reply:
Yes! I didn't want the name of the serializer to be too long, but I don't think that will be an issue (seeing as we already have a `PronounSeriesSerializer`). I've renamed it to `ProximityAnalysisSerializer`.